dmaws/stacks.py

```diff
@@ -195,7 +198,7 @@ def from_ctx(cls, ctx, **kwargs):
             variables=kwargs.get('variables', ctx.variables),
             apps=kwargs.get('apps', ctx.apps),
             logger=kwargs.get('logger', ctx.log),
-            profile_name=kwargs.get('profile_name'),
+            profile_name=kwargs.get('profile_name', ctx.stage)
```

Comment:
Setting a default here feels a bit scary, since it could mean that it will start using your production credentials when you don't expect it. Can we pass the profile_name in `dump_to_target` explicitly?